src/chi-vue/src/views/DataTable/ClientSide/DataTableClientView.vue

```diff
@@ -89,6 +89,62 @@
           </template>
         </ChiDataTableToolbar>
       </template>
+      <template #bulkActions>
+        <transition name="slide-fade">
+          <div v-show="bulkActionConfig.selectedRowIds.length > 0 && bulkActionConfig.showBulkActions">
+            <ChiDataTableBulkActions
+              :selectedRowCount="bulkActionConfig.selectedRowIds.length"
+              @showSelectedOnly="showSelectedOnlyRow"
```

Comment:
```suggestion
              @chiShowSelectedOnly="showSelectedOnlyRow"
```
For consistency, let's call the event `chiShowSelectedOnly`